Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -116,10 +108,16 @@ private ButtonInfo GatherCurrentInfo()
         {
             ButtonInfo info = new ButtonInfo();
 
-            info.TouchCageLocalBounds = new Bounds(touchCage.center, touchCage.size);
+            info.LocalCenter = touchable.LocalCenter;
+            info.PlaneExtents = touchable.Bounds;
 
-            Vector3 pressDirLocal = (touchable != null) ? -1.0f * touchable.LocalForward : Vector3.forward;
-            Vector3 upDirLocal = (touchable != null) ? touchable.LocalUp : Vector3.up;
+            Vector3 pressDirLocal = (touchable != null) ? touchable.LocalPressDirection : Vector3.forward;
+            Vector3 upDirLocal = Vector3.up;
+
+            if (touchable is NearInteractionTouchable touchableConcrete)
+            {
+                upDirLocal = touchableConcrete.LocalUp;
```

Comment:
should this be NearInteractionTouchableSurface?

Reply:
NearInteractionTouchableSurface has no LocalUp property.  It only has LocalCenter, LocalPressDirection, and Bounds.